sql/core/src/test/scala/org/apache/spark/sql/execution/command/ResourcesSuite.scala

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
How about naming `RunnableCommandSuite` instead of `ResourcesSuite`?

Reply:
Yeah. That's a better. I will rename the file.

Reply:
Or, it might be better to rename the existing `DDLSuite` to RunnnableCommandSuite` then add new tests there instead of creating a new test file. cc: @dongjoon-hyun

Reply:
I have renamed it to `RunnableCommandSuite` for the time being. 
@dongjoon-hyun Should i move the test to `DDLSuite` and rename it? or is it fine now only?